src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -77,15 +79,29 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   });
 
   const scheduledStartDate = sovereignVenue?.start_utc_seconds;
+  const scheduledEndDate = sovereignVenue?.end_utc_seconds;
 
-  const firstDayOfSchedule = useMemo(() => {
-    return scheduledStartDate
+  const { firstDayOfSchedule, isScheduleTimeshifted } = useMemo(() => {
+    const today = startOfToday();
+    const firstDayOfEvent = scheduledStartDate
       ? startOfDay(fromUnixTime(scheduledStartDate))
-      : startOfToday();
-  }, [scheduledStartDate]);
+      : today;
+
+    const isScheduleTimeshifted = isBefore(today, firstDayOfEvent);
 
-  const isScheduleTimeshifted = !isToday(firstDayOfSchedule);
+    const firstDayOfSchedule = isScheduleTimeshifted ? firstDayOfEvent : today;
 
+    return {
+      firstDayOfSchedule,
+      isScheduleTimeshifted,
+    };
+  }, [scheduledStartDate]);
+
+  const lastDayOfSchedule = useMemo(() => {
+    return scheduledEndDate
+      ? startOfDay(fromUnixTime(scheduledEndDate))
+      : undefined;
+  }, [scheduledEndDate]);
```

Comment:
[note] It doesn't matter that we used `startOfDay` here, as we only care about the date portion since we use `differenceInCalendarDays`; so the time within the day doesn't really matter.